src/vtex-ng-filter.html

```diff
@@ -53,8 +58,8 @@ <h3 class="group-header" ng-if="group && group.length > 0">
                 type="checkbox"
                 ng-model="filter.useTimezoneOffset"
                 ng-change="filter.onUseTimezoneOffsetChange()"
-                name="filter{{$index}}-use-timezone-offset"
-              > 
+                name="filter{{ $index }}-use-timezone-offset"
+              />
```

Comment:
maybe lint didn't get this one? (changed with line 125 ? 🤔)

Reply:
dunno 🤷‍♂️